### ISSUE TITLE
WWI-SalesOrders PBI Report Website.SalesOrders still missing

### ISSUE THREAD

**Issue by Kramomat** [Microsoft/sql-server-samples#496]
The issue has already been reported here:
https://github.com/Microsoft/sql-server-samples/issues/391

And is has been closed with a remark the table should exist in current Sourcecode version.
Unfortunately also the source code version is missing it. 
There are some stored procedures in that schema, but no tables at all.

Please check the code in, thanks

**Comment by FatherOSam** [comment]
I'm having similar issues - can the definition for this View be published by someone who has it ?

**Comment by dlytz** [comment]
Agreed, I am also missing this view after building and deploying from source using Visual Studio. Three other views are present, Website.Customers, .Suppliers and .VehicleTemperatures, but SalesOrders is absent as far as I can tell.

**Comment by prthor** [comment]
> The issue has already been reported here:
> #391
> 
> And is has been closed with a remark the table should exist in current Sourcecode version.
> Unfortunately also the source code version is missing it.
> There are some stored procedures in that schema, but no tables at all.
> 
> Please check the code in, thanks

I didn't find the views too, so I created them and posted it on the original thread. Let me know if it needs any changes.

**Comment by dwells-capstone** [comment]
Hello,

I found the DDLs for the view "SalesOrders.sql" "PurchaseOrderLines.sql" "PurchaseOrders.sql" "SalesOrderLines.sql" when are missing from the bacpac files for the WideWorldImporters database.

See here (https://github.com/jzebedee/sql-server-samples/tree/master/samples/databases/wide-world-importers/wwi-ssdt/wwi-ssdt/Website/Views).

I am looking to get these details rolled into this project to address a clear missing dependency issue.

Regards.

-DW
